plugins/hydra_ray_launcher/hydra_plugins/hydra_ray_launcher/ray_local_launcher.py

```diff
@@ -4,13 +4,13 @@
 from typing import Optional, Sequence
 
 import ray
-from omegaconf import DictConfig, open_dict
-
 from hydra import TaskFunction
 from hydra.core.config_loader import ConfigLoader
 from hydra.core.singleton import Singleton
 from hydra.core.utils import JobReturn, configure_log, filter_overrides, setup_globals
 from hydra.plugins.launcher import Launcher
+from omegaconf import DictConfig, open_dict
```

Comment:
Both OmegaConf and Hydra are third party libraries from the perspective of this plugin, so the sorting order here is wrong.
See isort [here](https://hydra.cc/docs/next/development/contributing#code-style-guide).